mobify/platform-scaffold
# Refactor recent develop features to fit in the integration manager branch

This PR refactors parts of the sort and filter features to better match the integration manager branch, and cleans up some errors.

 **JIRA**: N/A

## Changes
- Disable broken integration tests
- Fix broken unit test
- Hard-code sort options

## How to test-drive this PR
- (necessary config changes)
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Sort and filter on the PLP should still work.
